src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
 
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+
+    public static void getStatistic(String fromFileName, String toFileName) throws IOException {
+
+        BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        String line;
+        while ((line = reader.readLine()) != null) {
+            String[] fields = line.split(",");
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
            String[] fields = line.split(SEPARATOR);
```